pyproject.toml

```diff
@@ -19,8 +19,9 @@ requires-python = ">=3.9"
 dynamic = ["version"]
 
 dependencies = [
-    "mqt.bench>=0.4.2,<0.4.3",
-    "sb3_contrib>=1.6.2",
+    "mqt.bench>=0.4.4,<0.4.5",
+    "qiskit>=0.42.0",
```

Comment:
that should not be necessary as it is inherently covered by `mqt.bench` and `pytket-qiskit`.
If this really is a hard lower bound, then the lower bound of the `pytket-qiskit` dependency in `mqt.bench` should be adjusted.

Reply:
You are right, this is only an intermediate fix until we update `mqt.bench` rather soon after the new version is released. The reason behind is, that we need the `.layout` attribute of `Qiskit.QuantumCircuit` for the RL part.

Therefore, I suggest to keep it as is for now and set a reminder to remove it later.